packages/cli/src/commands/setup/auth/templates/dbAuth.auth.ts.template

```diff
@@ -49,11 +48,17 @@ type AllowedRoles = string | string[] | undefined
  * @returns {boolean} - Returns true if the currentUser is logged in and assigned one of the given roles,
  * or when no roles are provided to check against. Otherwise returns false.
  */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
@cannikin headsup - I'm commenting out the roles checks here, and leaving a comment to explain how they could use it. DB auth doesn't come out of the box with roles configured, especially because String[] aren't supported on sqlite, it might be best to leave it out.

Reply:
@dac09 I believe there's a PR open for this already. @dthyresson was taking a look since he originally commented out

Reply:
We want `roles` to just start magically working if/when you add `User.roles` and this code being in place makes that happen! Here's the PR we just merged that re-enabled these checks: https://github.com/redwoodjs/redwood/pull/4476

Reply:
Its because TypeScript correctly won't let you do that. It knows that `currentUser.roles` doesn't exist, thanks to Prisma's types. 

So our choices here are either:
**a) We ship with roles**
which means that it'll magically work when you add roles (but only on after you switch to postgres/mysql/etc.) 
BUT, setup dbauth will be creating code with typescript errors (and I'll need to hack stuff in e2e, so that it ignores/removes the roles code)

or 

**b) We leave it commented** 
which means the setup command creates code that will pass `yarn rw type-check` and e2e tests.
- BUT it won't magically work anymore, and will require them to uncomment when going through the tutorial to add RBAC. 

On another note, isn't it a good thing that they go into the code and uncomment? Kinda useful to understand how that part of the code works, no?

Reply:
@cannikin @dac09 

Here's what I suggest:
- leave dbAuth template as is with `roles` uncommented
- this will create a TS error, which we can handle for this test by commenting out (via codemode during rebuildFixture)

Reply:
@cannikin and I have found a compromise. I will update this in the morning!

Reply:
@dac09 I had to revert your changest to the fixture/test-project because the migrations were throwing the Smoke-test

Just FYI that the type-check will throw on Auth right now